proxy/pkg/config/config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	ReplaceCqlFunctions     bool   `default:"false" split_words:"true"`
 	AsyncHandshakeTimeoutMs int    `default:"4000" split_words:"true"`
 	LogLevel                string `default:"INFO" split_words:"true"`
+	ProtocolVersion         uint   `default:"3" split_words:"true"`
```

Comment:
This should be `ControlConnMaxProtocolVersion` to make it clear we are not messing with the protocol version on the non control connections. And this setting is confusing because at the moment it is more of a "MaxProtocolVersion" instead of one that is "forced" which I as a user would expect (I think) if I just read the name "Protocol Version".